GoogleForCreators/web-stories-wp
# Dashboard: My Stories Search (part 1)

# Overview 
Initial implementation of search for my stories. My thought is that this structure will allow easy reuse of the search component for the other dashboard views. 
- ONLY filters by title right now. Metadata will come later
- There is no 'no results' view yet 
- Pills have not been accounted for in story quantity by filtering

Filters stories in display as you type in your search term. Currently filters on story title. Clearing search will return all your stories to view 

## Testing 
- run locally & navigate to dashboard 'my stories' 
- see the search component 
- type into the search component and see your stories filter appropriately based on the story title 
- clearing your search restores all of your stories to view 
- play with your viewport size, see that the search component shrinks to the search icon and will expand when clicked on and still work the same as documented above. 

![search demo](https://user-images.githubusercontent.com/10720454/78165259-e8ee2f80-73ff-11ea-9306-ecffa66b383b.gif)
